lib/net.c

```diff
@@ -451,7 +451,7 @@ lua_free_iface(struct gatekeeper_if *iface)
 	destroy_iface(iface, IFACE_DESTROY_LUA);
 }
 
-static int
+int
 get_ip_type(const char *ip_addr)
```

Comment:
Whenever `get_ip_type()` returns error, it should return `AF_UNSPEC` instead of `1` because `1 == AF_LOCAL`.